doc/source/advanced/datatype.rst

```diff
@@ -0,0 +1,90 @@
+***********************************
+Data Types in Lux
+***********************************
+
+.. note:: You can follow along this tutorial in a Jupyter notebook. 
+
+Currently, Lux supports four data types:
```

Comment:
This is a bit abrupt, start by introducing what are data types and why are data types important in Lux.

Reply:
This seems like a good spot to give a brief overview on how Lux uses these data types for chart generation.